system_files/deck/shared/usr/share/ublue-os/just/85-bazzite-image.just

```diff
@@ -144,6 +144,39 @@ install-speaker-firmware ACTION="":
     fi
     rm /tmp/bazzite-speaker.sh
 
+# Configure video overrides for Steam
+configure-override-videos ACTION="":
+    #!/usr/bin/bash
+    source /usr/lib/ujust/ujust.sh
+    OPTION={{ ACTION }}
+
+    if [ ! -f $HOME/.local/share/Steam/config/uioverrides/movies/bazzite_novideo ]; then
+        VIDEO_STATE="${b}Enabled${n}"
+    else
+        VIDEO_STATE="${b}Disabled${n}"
+    fi
+
```

Comment:
i would suggest avoiding empty lines like this when writing bash (or other script languages) in justfiles, the linter sometimes really hates it. and the just syntax highlighters also dislike the empty lines a lot :sweat_smile: